src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -79,6 +79,7 @@
   protected static final By LOGIN_BUTTON_CSS = By.cssSelector("a[data-id='login']");
   private static final String LOGGED_IN_USER_SELECTOR_OASIS =
       ".AccountNavigation a[title*=%userName%]";
+  private static final By MERCURY_SKIN = By.cssSelector("#preload .site-head, .site-head .site-head-icon-nav");
```

Comment:
Uhm... No, it's not :D But there is `#ember-container`! And this one is static too.